sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/TrainingOperation.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.DocumentIntelligence
+{
+    internal class TrainingOperation : Operation<BinaryData>
```

Comment:
Not a blocker, but I wonder if it's possible to auto-generate this somehow.

Reply:
Not sure. This is a workaround that Michael suggested while the `Operation.Id` support is not added to the generator. Hopefully we can get it before GA.